### PR TITLE
Makes changeling regenerate power description slightly more descriptive

### PR DESCRIPTION
**Pull request by Yenwodyah**
## Why It's Good For The Game
Cause I didn't know that regenerate healed wounds until I checked the code just now, and it really would have saved my ass if I'd known this in a ling round I played earlier today...

### CODE REVIEW

File: code/modules/antagonists/changeling/powers/regenerate.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /datum/action/changeling/regenerate
 	name = "Regenerate"
-	desc = "Allows us to regrow and restore missing external limbs and vital internal organs, as well as removing shrapnel and restoring blood volume. Costs 10 chemicals."
+	desc = "Allows us to regrow and restore missing external limbs and vital internal organs, as well as removing shrapnel, healing all wounds, and restoring blood volume. Costs 10 chemicals."
```
Comment (ATH1909) [inline code/modules/antagonists/changeling/powers/regenerate.dm:3]: you need to specify that this is in reference to "wounds" as in the SS13 game mechanic, not "wounds" as in the synonym for damage, or else new changeling players are gonna think that this is fleshmend lite.

Comment (tralezab) [reply code/modules/antagonists/changeling/powers/regenerate.dm:3]: Oh please

Comment (Fikou) [reply code/modules/antagonists/changeling/powers/regenerate.dm:3]: sjhut he fuck up[

Comment (Winterous) [reply code/modules/antagonists/changeling/powers/regenerate.dm:3]: It's a reasonable concern, but I can't think of a way to word it which both specifies that information and isn't super clumsy.

Comment (Winterous) [reply code/modules/antagonists/changeling/powers/regenerate.dm:3]: Honestly I don't see a reason that healing wounds shouldn't be relegated to fleshmend in the first place, but eh.

Comment (Yenwodyah) [reply code/modules/antagonists/changeling/powers/regenerate.dm:3]: ok, how about "healing major wounds" instead?

Comment (MrDoomBringer) [reply code/modules/antagonists/changeling/powers/regenerate.dm:3]: yes, healing major wounds works better

**Comment by MrDoomBringer** [comment]
btw S tier branch name

**Review by Rohesie — APPROVED** [review]

**Pull request merged**